certbot/certbot/display/ops.py

```diff
@@ -19,17 +19,19 @@
 logger = logging.getLogger(__name__)
 
 
-def get_email(invalid: bool = False) -> Optional[str]:
+def get_email(invalid: bool = False, optional: bool = True) -> str:
```

Comment:
You could do a `kwargs` thing here I think? Should be the same in practice, just ignoring anything else. Either way works for me.